boards/FSM/Src/Io/Io_AcceleratorPedals.c

```diff
@@ -4,9 +4,11 @@
 
 bool Io_AcceleratorPedals_IsPappsEncoderAlarmActive(void)
 {
-    return HAL_GPIO_ReadPin(
-               PRIMARY_APPS_ALARM_GPIO_Port, PRIMARY_APPS_ALARM_Pin) ==
-           GPIO_PIN_SET;
+    //    return HAL_GPIO_ReadPin(
+    //               PRIMARY_APPS_ALARM_GPIO_Port, PRIMARY_APPS_ALARM_Pin) ==
+    //           GPIO_PIN_SET;
+
```

Comment:
```suggestion

```

Reply:
lets clean this up in a future PR, can you make an issue for this? @gtaharaedmonds